### DISCUSSION TITLE
Atom parou no meio da instalação.

### DISCUSSION THREAD

**Discussion by Adriano-Web** [frontendbr/forum#1181] [Discussão]
Estou com um problema na instalação do Atom 1.29.0.0, ele para no meio da instalação e me apresenta uma janela com um circulo com um x vermelho. Ele me apresentou um log de amostra do erro, podem identifica-lo? Abaixo deixo-o para que o analisem-no.

```
CheckForUpdateImpl: Couldn't write out staging user ID, this user probably shouldn't get beta
anything: System.IO.DirectoryNotFoundException: Não foi possível localizar uma parte do caminho
'C:\Users\Positivo\AppData\Local\atom\packages.betaId'.
```

**Comment by vitor-mendrone-deel** [comment]
@Adriano-Web Tentou postar essa dúvida direto no Stack Overflow ou no próprio repositório do Atom? 

Vou editar a issue pra deixar mais curta, minha sugestão seria você postar esse stack trace no pastebin ou em um gist e postar o link aqui, é mais provável que você consiga ajuda dessa forma.
